editor/src/clj/editor/protobuf.clj

```diff
@@ -55,6 +55,9 @@ Macros currently mean no foreseeable performance gain however."
       (.getBytes StandardCharsets/UTF_8)
       (TextFormat/escapeBytes)))
 
+(defn pb-class? [value]
+  (isa? value Message))
```

Comment:
`isa?` might be too open, i.e. it will return true both for subclasses of Message and values that are instances of the class. Maybe use something like `(.isAssignableFrom Message value)`?

Reply:
Good point! We also do not want to allow the `Message` class itself here. I'll fix!